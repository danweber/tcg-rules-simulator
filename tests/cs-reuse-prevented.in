VERSION:2
#HAND2:CS3-02

# if my effect is prevented, I can still use it later

FIELD2:Roger,CS3-Chirp Vic1,DW1-Van,REST Vic2,DW1-Van,REST
FIELD1:Deleter,DW1-Big,CS2-05
EGGZONE1:CS3-01
HAND1:DW1-Standup


GAME:P1 T4 MAIN 8
GET-X-PLAYS
DUMP:HAND,KEYWORDS

ECHO: this test can fail if we haven't loaded up <Dodge> from another card, like cs3-playfromstack

CMD:ATTACK Deleter Vic1
STEP:AUTOANSWER:CAPTURE

CMD:PLAY DW1-Standup
DUMP

ECHO: ---- try again ----

CMD:ATTACK Deleter Vic2
STEP:AUTOANSWER:CAPTURE

DUMP


EXIT
