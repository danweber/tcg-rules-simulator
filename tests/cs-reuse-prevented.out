GET-X-PLAYS: json REFRESH
GET-X-PLAYS: PLAY 0 Use Standup
GET-X-PLAYS: ATTACK 4 2 Attack BigMon into VanillaMon
GET-X-PLAYS: ATTACK 4 3 Attack BigMon into VanillaMon
GET-X-PLAYS: ATTACK 4 0 Attack BigMon into player
GET-X-PLAYS: NEXT PASS TURN
FIELD1:Deleter,DW1-Bigmon,CS2-Greatsword
FIELD2:Roger,CS3-Chirpflare,DODGE Vic1,DW1-Vanillamon,REST Vic2,DW1-Vanillamon,REST
HAND1:DW1-Standup
HAND2:
GAME:P1 T4 MAIN 8
ECHO: this test can fail if we haven't loaded up <Dodge> from another card
CAPTURE:Activate ChirpFlare suspending
CAPTURE:Don't pay/activate
FIELD1:Deleter,DW1-Bigmon,CS2-Greatsword
FIELD2:Roger,CS3-Chirpflare,REST Vic2,DW1-Vanillamon,REST
GAME:P1 T4 MAIN 4
ECHO: ---- try again ----
FIELD1:Deleter,DW1-Bigmon,CS2-Greatsword,REST
FIELD2:
GAME:P1 T4 MAIN 4
